dali/pipeline/operator/op_schema.h

```diff
@@ -388,6 +411,75 @@ class DLL_PUBLIC OpSchema {
 
   DLL_PUBLIC string Dox() const;
 
+  DLL_PUBLIC bool CanUseDefaultInputDox() {
+    // TODO(klecki): add a flag to disallow this
+    if (MinNumInput() == 1 && MaxNumInput() == 1 && input_docs_.size() == 0) {
+      return true;
+    }
+    return false;
+  }
+
+  /**
+   * @brief Check if this operator has input documentation specified
+   *
+   * If this is true `GetCallSignatureInputs` and `InputDox` can be used to obtain it
+   *
+   * @return DLL_PUBLIC HasInputDox
+   */
+  DLL_PUBLIC bool HasInputDox() {
```

Comment:
```suggestion
  DLL_PUBLIC bool HasInputDocs() {
```

Reply:
Ask Przemek why he has chosen the `Dox` instead of `Docs`

Reply:
@ptrendx ?